cylc/flow/network/server.py

```diff
@@ -324,7 +324,7 @@ def graphql(
             executed: ExecutionResult = schema.execute(
                 request_string,
                 variable_values=variables,
-                context={
```

Comment:
Internal deprecation within Graphene?

Reply:
At which version?

Reply:
@MetRonnie just checking that this doesn't remove compatibility with our specified version range before merging.

Reply:
I have done some digging; we are pinned at `graphene>=2.1,<3`, and graphene 2.1.0 is pinned at `graphql-core>=2.0,<3`. I have looked through graphql-core 2.0.0 and only the `context_value` arg is used. (This was deprecated in favour of `context` before they changed their minds and made `context` deprecated in favour of `context_value` in between 2.0.0 and the latest 2.x release)

Reply:
Ok, that's strange!